### PR TITLE
changed flex to grid

### PR DESCRIPTION
**Pull request by silchyn**
- [DEMO LINK](https://silchyn.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by OlgaStakhova — COMMENTED** [review]
Згідно сhecklist зауважень не маю.

File: src/styles/main.scss [hunk 1]
```diff
@@ -10,5 +10,7 @@ body {
   box-sizing: border-box;
 }
 
+@import "_variables";
+
 @import "_header.scss";
```
Comment (ivan-kadykalo) [inline src/styles/main.scss:15]: Where is this file?

**Review by ivan-kadykalo — APPROVED** [review]
👍

File: src/styles/main.scss (hunk 1 above)
Comment (silchyn) [reply src/styles/main.scss:15]: In the `/styles/`

**Pull request closed without merging**